### PR TITLE
Feature/eb basic styling

### PR DESCRIPTION
**Pull request by elisebeall**
## [Rancid Tomatillos]

- [x] - Feature PR [🧩]
- [ ] - Refactor PR [⚙️]
- [ ] - BugFix PR [🐞]
- [x] - User Interface PR [🖍]
- [ ] - Error Handling PR [⚠️]
- [ ] - Testing PR [🧑‍💻]

**What (if any) features are you implementing?**

 - For this pull request, the focus was on displaying mock data in a logical fashion on two different pages, as well as creating a navigable page for the user. This merge will close the following issues:
 - #2 
 - #5 
 - #7 
 - #8 
 - #12
 - #15

**What (if anything) did you refactor?**

 - (NA)

**Were there any issues that arose?**

 - (NA)

**Is there anything needs to be prioritized/errors to be addressed?**

 - (NA)

**Goals? Next steps?**

 - For the user interface, the goal will be to refine the styling by including padding and margins.  Adding icons & styling to buttons and containers.  
 - To add more features & functionality, the next steps will be to implement the API fetch calls to bring in more data.  This will give us endpoints for each of the individual movies, as well as background images & trailers.

screen shot of home page

screen shot of movie details

### CODE REVIEW

**Comment by elisebeall** [comment]
@kaylagordon @CassandraGoose

File: src/index.css [hunk 1]
```diff
@@ -11,3 +11,6 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* @import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@1,300&display=swap'); */
```
Comment (annamkummer) [inline src/index.css:15]: Can these comments be removed?

File: src/css/Poster.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+img {
+  margin: 10px;
+  /* border: 2px solid #594600; */
+  height: 200px;
+}
+
+img:hover {
```
Comment (annamkummer) [inline src/css/Poster.css:7]: I really like this feature!

**Review by annamkummer — CHANGES_REQUESTED** [review]
Please remove commented out sections, thanks!

**Comment by elisebeall** [comment]
Closes #2

**Review by elisebeall — COMMENTED** [review]
Changes made!

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## [name of project]
```
Comment (elisebeall) [inline PULL_REQUEST_TEMPLATE.md:1]: This is actually just a test comment.  Please pretend that you never even saw it.

**Review by elisebeall — COMMENTED** [review]
Closes #5
Closes #7 
Closes #15

File: src/components/App.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+// import { render } from '@testing-library/react';
```
Comment (annamkummer) [inline src/components/App.js:1]: Can we remove this comment?

**Review by annamkummer — COMMENTED** [review]
Remove comment?

File: src/components/App.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import PosterGrid from './PosterGrid'
+import MovieDetails from './MovieDetails'
+import movieData from '../movieData'
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      moviePosters: movieData.movies,
+      showAllMovies: true,
```
Comment (kaylagordon) [inline src/components/App.js:14]: You'll be able to cut this once Router is implemented

File: src/components/Poster.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import '../css/Poster.css'
+
+const Poster = (props) => {
```
Comment (kaylagordon) [inline src/components/Poster.js:3]: Consider destructuring your props so you don't have to write `props.` so many times

**Review by kaylagordon — COMMENTED** [review]
Looks great so far! I only had minor comments. My biggest feedback is that this is A LOT of code changes for one PR. I would consider this to be enough changes/features to be 3+ PRs. Consider pushing more often!

**Comment by elisebeall** [comment]
> Looks great so far! I only had minor comments. My biggest feedback is that this is A LOT of code changes for one PR. I would consider this to be enough changes/features to be 3+ PRs. Consider pushing more often!

Thanks @kaylagordon!

It _was_ a big pull request, but we specifically wanted to wait to merge with main until we had something that we felt showed all of the basic pieces.  @annamkummer created a branch for our original component architecture, and another for the movie details page.  I created a branch for styling.  From inside my styling branches, I would pull down Anna's branches and **attempt** to make them look pretty!  We definitely could have done it in smaller steps, but then main would've been something not ready for viewing.  Does that make sense?  I'd never been good about sticking to it in the past, but my understanding is, the main branch should be 'production ready'.

Thanks again for the code review! 🥰

**Pull request merged**